### PR TITLE
add 'Release claim' button

### PR DESCRIPTION
**Pull request by liewegas**
image

### CODE REVIEW

File: src/slack_interaction_handler.ts [hunk 1]
```diff
@@ -348,15 +355,17 @@ export async function handleVolunteerUpdate({
...
+      : `*Operator:* Volunteer claim released by *${originatingSlackUserName}* at *${specialSlackTimestamp}*.`,
     {
       parentMessageTs: payload.container.thread_ts,
       channel: payload.channel.id,
     }
   );
 
   logger.info(
-    `SLACKINTERACTIONHANDLER.handleVoterStatusUpdate: Successfully sent message recording voter status change`
+    `SLACKINTERACTIONHANDLER.handleVolunteerUpdate: Successfully sent message recording volunteer status change`
```
Comment (tomerovadia) [inline src/slack_interaction_handler.ts]: nit: can we do "volunteer claim" instead of "volunteer status"?

File: src/slack_interaction_handler.ts [hunk 2]
```diff
@@ -348,15 +355,17 @@ export async function handleVolunteerUpdate({
 
   // Post a message in the voter thread recording this status change.
   await SlackApiUtil.sendMessage(
-    `*Operator:* Volunteer changed to *${selectedVolunteerSlackUserName}* by *${originatingSlackUserName}* at *${specialSlackTimestamp}*.`,
+    selectedVolunteerSlackUserName
+      ? `*Operator:* Volunteer changed to *${selectedVolunteerSlackUserName}* by *${originatingSlackUserName}* at *${specialSlackTimestamp}*.`
+      : `*Operator:* Volunteer claim released by *${originatingSlackUserName}* at *${specialSlackTimestamp}*.`,
```
Comment (tomerovadia) [inline src/slack_interaction_handler.ts:360]: Is `originatingSlackUserName` null here in the release claim case?

Comment (liewegas) [reply src/slack_interaction_handler.ts:360]: no, originatingSlackUserName is the person who clicked the button (not necessarily the old assigned volunteer)

Comment (liewegas) [reply src/slack_interaction_handler.ts:360]: I wasn't certain if the payload's value for the drop-down would reliably have the old volunteer so i left it off the message. I'm not sure it is that important.. you can look in the history to see who the assigned person was before

Comment (tomerovadia) [reply src/slack_interaction_handler.ts:360]: Right thanks! No problem, I don't think there's a need to have the message include the old volunteer claim.

File: src/slack_block_util.ts [hunk 3]
```diff
@@ -136,6 +136,17 @@ const volunteerSelectionPanel: SlackBlock = {
         emoji: true,
       },
     },
+    {
+      type: 'button',
+      style: 'primary',
+      text: {
+        type: 'plain_text',
+        text: 'Release claim',
```
Comment (tomerovadia) [inline src/slack_block_util.ts]: I wonder if "Clear volunteer" is clearer for the button text

**Review by tomerovadia — COMMENTED** [review]
Can you add a screenshot to the description?

Also can you ask if Andrew can review? Feel free to QA in the meantime.

File: src/slack_block_util.ts (hunk 3 above)
Comment (liewegas) [reply src/slack_block_util.ts]: updated!

**Review by tomerovadia — APPROVED** [review]

**Pull request merged**